src/plugins/selected-suggestion-ui.ts

```diff
@@ -146,6 +146,7 @@ export default (props: SelectedSuggestionProps) =>
 
         if (!trackChangesData) {
           // if running dispatch synchroniously it adds about 30ms in this case to the event capturing which results in corresponding delay for the event handler
+          // IMPORTANT: this causes issues with the context menu, we need to find a way to run this synchronously
```

Comment:
This plugin will be removed with changes in https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/273